source/appModules/putty.py

```diff
@@ -23,4 +23,4 @@ def chooseNVDAObjectOverlayClasses(self, obj, clsList):
 				clsList.remove(DisplayModelEditableText)
 			except ValueError:
 				pass
-			clsList[0:0] = (Terminal, DisplayModelLiveText)
+			clsList[0:0] = (TerminalWithKeyboardSupport, DisplayModelLiveText)

```

Comment:
I think it should do this only if on windows 10  build 1607+ right?

Reply:
@feerrenrut I'm not sure if we really need to check here. Since Windows versions below 1607 don't support any of the new functionality, the object should just work as-is (it's not like we're breaking any existing keyboard support).